.github/workflows/smoke-checks.yml

```diff
@@ -92,82 +91,84 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: ./.github/actions/bootstrap
-    - name: "Prepare ${{ env.IOS_SIMULATOR_DEVICE }} with iOS ${{ env.IOS_VERSION }}"
-      uses: ./.github/actions/prepare-ios-simulator
-      with:
-        device: ${{ env.IOS_SIMULATOR_DEVICE }}
-        ios: ${{ env.IOS_VERSION }}
-        xcode: ${{ env.XCODE_VERSION }}
+    - name: Setup iOS ${{ matrix.ios }} Runtime
+      run: xcversion simulators --install='iOS ${{ env.IOS_VERSION }}'
     - name: Run UI Tests (Debug)
       run: bundle exec fastlane test_ui device:"${{ env.IOS_SIMULATOR_DEVICE_WITH_VERSION }}"
       timeout-minutes: 25
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         GITHUB_EVENT: ${{ toJson(github.event) }}
+    - uses: actions/upload-artifact@v3
+      if: failure()
+      with:
+        name: snapshots
+        path: |
+          Tests/StreamChatUITests/SnapshotTests/ChatChannel/__Snapshots__/ChatChannelVC_Tests/test_whenMessageFromCurrentUserIsFollowedByEphemeralMessage_messagesAreNotGrouped.default-light.png
```

Comment:
Is this only for testing purposes?

Reply:
Yeah, debugging failing snapshot test on CI